.gitignore

```diff
@@ -7,6 +7,7 @@ venv/
 __pycache__/
 # IDEs
 .idea/
+.vscode/
```

Comment:
I'm genuinely surprised we've managed to get this far without adding `.vscode` to the gitignore. Thanks for adding it here.